.browserslistrc

```diff
@@ -0,0 +1,12 @@
+node >= 10
+last 2 Chrome versions
+last 2 Edge versions
+last 2 Firefox versions
+last 2 Safari versions
+last 2 Opera versions
```

Comment:
We don't run Opera in our CI tests, right? Do you want to list Opera anyway?
This list could be added instead to our _package.json,_ easier accessable for our users.

Do we need `node >= 10`?

Reply:
For browsers, no. But in case we're some day in a situation where our oldest node version has less capabilities than our oldest browser support, it can be useful for linting plugins.

I think I'll need to give the browserslist support range another do-over later anyway, based on a discussion we need to have about transpilation and polyfills